app/views/claimToAdjustPoa/ConfirmationForAdjustingPoa.scala.html

```diff
@@ -0,0 +1,73 @@
+@*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import views.html.components._
+@import views.html.layouts.unifiedLayout
+@import uk.gov.hmrc.govukfrontend.views.html.components._
+@import models.incomeSourceDetails.TaxYear
+
+@this(
+        mainTemplate: unifiedLayout,
+        appConfig: config.FrontendAppConfig,
+        h1: h1,
+        h2: h2,
+        p: p,
+        link: link,
+        form: FormWithCSRF,
+        govukBackLink : GovukBackLink,
+        govukButton : GovukButton
+)
+
+@(isAgent: Boolean, poaTaxYear: TaxYear, isAmountZero: Boolean)(implicit messages: Messages, request: RequestHeader, user: auth.MtdItUser[_])
```

Comment:
If we can create a view model for this view pls
something like: 
`case class ConfirmationForAdjustingPoa(poaTaxYear: TaxYear, isAmountZero: Boolean) // maybe to include isAgent: Boolean ? `